js-functions-and-switch/js-homework-3.js

```diff
@@ -0,0 +1,30 @@
+const zipCode = (zip) =>{
+    if(zip>=1000 && zip<=7000){
+        return true;
+    }
+    else{
+        return false;
+    }
+}
+const printMessage1 = zipCode(5000);
+console.log(printMessage1);
+
+
+const anotherFunction = (name,country, zip ,mobileNumber) =>{
+    printMessage1;
+    if(printMessage1){
+        console.log(`Hello ${name}, Your data: Country - ${country}, Zip code - ${zip}, Mobile number - ${mobileNumber}`);
+    }
+    else{
+        console.log('Registration Failed. Zip code is not valid');
+    }
+}
+const printMessage2 = anotherFunction('Sara','Macedonia', '5000' ,123456);
+console.log(printMessage2);
```

Comment:
`undefined` in the console comes from this console.log.
`anotherFunction` is void function and it is not returning a value (you are just using console.log, and not a return), so you don't have a value to be stored in the `printMessage2` constant at the line above.

- If you want to make this console.log working, you need to use `return` instead of `console.log` in `anotherFunction` body, on the lines 16 and 19.

- Another way to get rid of `undefined` will be to remove this `console.log` because you are already printing the messages from the `anotherFunction` body, and on line 22, you can just call the function without storing it into a variable:
`anotherFunction('Sara','Macedonia', '5000' ,123456);` 
instead of:
`const printMessage2 = anotherFunction('Sara','Macedonia', '5000' ,123456);`